include/zenoh-pico/system/platform.h

```diff
@@ -31,6 +31,8 @@
 #include "zenoh-pico/system/platform/arduino/esp32.h"
 #elif defined(ZENOH_ARDUINO_OPENCR)
 #include "zenoh-pico/system/platform/arduino/opencr.h"
+#elif defined(ZENOH_WASM)
```

Comment:
What about changing `ZENOH_WASM` to `ZENOH_EMSCRIPTEN` since this port is tied to Emscripten as the compiler toolchain for WebAssembly?